src/main/java/MariaDbLauncher.java

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
cc @ibacher @dkayiwa , back on this ticket, could you take a review about this class